chore: clean up taxonomy on hosted-specific pages (#2400)
mendersoftware/mender-docs: 08.Server-integration/03.Mutual-TLS-authentication/01.Keys-and-certificates/docs.md

export MTLS_IP=

> # You need to create a separate user on your Mender instance.

Reviewer: ```suggestion
# You need to create a separate user in your Mender account.
```
